src/ontology/cl-edit.owl

```diff
@@ -32721,6 +32723,25 @@ AnnotationAssertion(rdfs:label obo:CL_4042034 "CGRP-expressing neuron"@en)
 SubClassOf(obo:CL_4042034 obo:CL_0000526)
 SubClassOf(obo:CL_4042034 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001017))
 
+# Class: obo:CL_4042035 (molecular layer interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30742002") Annotation(oboInOwl:hasDbXref "PMID:33075461") Annotation(oboInOwl:hasDbXref "PMID:34616064") Annotation(oboInOwl:hasDbXref "PMID:38692278") obo:IAO_0000115 obo:CL_4042035 "A type of cerebellar inhibitory GABAergic interneuron that is located in the molecular layer of the cerebellum. This cell type inhibits Purkinje cells and other molecular layer interneurons.")
+AnnotationAssertion(terms:contributor obo:CL_4042035 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042035 "2024-10-31T15:55:31Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33075461") Annotation(oboInOwl:hasRelatedSynonym obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4042035 "MLI")
+AnnotationAssertion(rdfs:label obo:CL_4042035 "molecular layer interneuron"@en)
+SubClassOf(obo:CL_4042035 obo:CL_1001611)
+
+# Class: obo:CL_4042036 (melanin-concentrating hormone neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30284033") Annotation(oboInOwl:hasDbXref "PMID:38020069") obo:IAO_0000115 obo:CL_4042036 "A neuron of the central nervous system with its soma located in the posterior lateral hypothalamus, this neuron type expresses melanin-concetratic hormone.")
```

Comment:
Consider adding some functions (PMID: 39035036):

> With extensive projections throughout various brain areas, MCH neurons are part of the neural network that regulates many physiological processes, including energy homeostasis (2–4), sleep and wakefulness (5–7), reward and reinforcement feeding (8), memory, and learning (9).

Consider adding these info to the definition: 

A neuron of the central nervous system with its soma primarily located in the lateral hypothalamic area and surrounding regions, including the zona incerta. This neuron type expresses melanin-concentrating hormone and is involved in regulating various physiological processes, including sleep, feeding behavior, and energy homeostasis.